src/app/pages/PerpetualPage/utils/bscWebsocket.ts

```diff
@@ -0,0 +1,56 @@
+import Web3 from 'web3';
+import IPerpetualManager from 'utils/blockchain/abi/PerpetualManager.json';
+import { AbiItem } from 'web3-utils';
+
+// TODO: Change subscription ID to perpID + candleDuration
+
+const web3Socket = new Web3(
+  new Web3.providers.WebsocketProvider(
+    'wss://ws-nd-233-405-699.p2pify.com/' +
```

Comment:
@ShynRou I've hardcoded the testnet websocket in here, but it should be moved to a config file somewhere with the mainnet websocket. I wasn't sure where you wanted to store testnet/mainnet rpc urls